sdk/cosmos/azure-cosmos/src/samples/java/com/azure/cosmos/ContainerAsyncCodeSnippets.java

```diff
@@ -0,0 +1,152 @@
+package com.azure.cosmos;
+
+import com.azure.cosmos.models.*;
+import reactor.core.publisher.Mono;
+
+import java.util.Collections;
+import java.util.List;
+
+public class ContainerAsyncCodeSnippets {
+    private final String serviceEndpoint = "<service-endpoint>";
+    private final String key = "<key>";
+    private final DirectConnectionConfig directConnectionConfig = new DirectConnectionConfig();
+    private final GatewayConnectionConfig gatewayConnectionConfig = new GatewayConnectionConfig();
```

Comment:
looks like these connection configs are not used.

Reply:
Fixed it